docs/docs/noir/concepts/generics.md

```diff
@@ -104,3 +104,49 @@ impl Eq for MyStruct {
 ```
 
 You can find more details on traits and trait implementations on the [traits page](../concepts/traits.md).
+
+## Turbofish
+
+Paths with generic parameters in expressions must prefix the opening brackets with a `::<>` operator.
+The name "turbofish" comes from that `::<>` looks like a little fish.
```

Comment:
We should explain what "turbofish" is and what it is for first, this description just jumps right it.

Perhaps we should also rename the section to something like "Manually Specifying Generics with the Turbofish Operator" or something more descriptive.

Reply:
Ok I'll expand the description

Reply:
What do you think of the blurb I have added?